commenting/.babelrc

```diff
@@ -1,3 +1,11 @@
 {
-  "presets": [ "env" ]
-}
\ No newline at end of file
+  "presets": [
+    [
+      "env", {
+        "targets": {
+          "node": "current"
```

Comment:
What is this about? Why is it needed?

Reply:
Because of this
<img width="789" alt="Screenshot 2020-05-28 at 13 10 18" src="https://user-images.githubusercontent.com/15003836/83134402-99b53a00-a0e4-11ea-92ef-65933a16f85c.png">

Now that we're using async keyword, we need to configure babel accordingly: https://github.com/babel/babel/issues/8829